diff --git a/dist/react-datepicker.js b/dist/react-datepicker.js
index ae82bcf47146deeb250b7cc62c4d89e0bed5068a..42c41972c77433c14bfb465c0b50a7d0d78c8bf4 100644
--- a/dist/react-datepicker.js
+++ b/dist/react-datepicker.js
@@ -86,10 +86,11 @@
         onClickOutsideRef.current = onClickOutside;
         var handleClickOutside = React.useCallback(function (event) {
             var _a;
-            if (ref.current && !ref.current.contains(event.target)) {
+            var target = (event.composed && event.composedPath && event.composedPath().find(eventTarget => eventTarget instanceof Node)) || event.target;
+            if (ref.current && !ref.current.contains(target)) {
                 if (!(ignoreClass &&
-                    event.target instanceof HTMLElement &&
-                    event.target.classList.contains(ignoreClass))) {
+                    target instanceof HTMLElement &&
+                    target.classList.contains(ignoreClass))) {
                     (_a = onClickOutsideRef.current) === null || _a === void 0 ? void 0 : _a.call(onClickOutsideRef, event);
                 }
             }
diff --git a/package.json b/package.json
index 0e83c941ec3b2ce1cb7d3edba075c4a9d870b2f0..e21c676a2a7dd51cfe0b0e1c4a8b25eb07abf536 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "homepage": "https://github.com/Hacker0x01/react-datepicker",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "browser": "dist/react-datepicker.min.js",
+  "browser": "dist/react-datepicker.js",
   "module": "dist/es/index.js",
   "unpkg": "dist/react-datepicker.min.js",
   "style": "dist/react-datepicker.min.css",
