diff --git a/dist/generateDocgenCodeBlock.js b/dist/generateDocgenCodeBlock.js
index 03477dda70177ba884a3f8b494a0cc8783bf9ca0..2742e82f042bac9a18fcbeeb37ea11f884a78da1 100644
--- a/dist/generateDocgenCodeBlock.js
+++ b/dist/generateDocgenCodeBlock.js
@@ -35,7 +35,7 @@ function insertTsIgnoreBeforeStatement(statement) {
  * ```
  */
 function setDisplayName(d) {
-    return insertTsIgnoreBeforeStatement(typescript_1.default.factory.createExpressionStatement(typescript_1.default.factory.createBinaryExpression(typescript_1.default.factory.createPropertyAccessExpression(typescript_1.default.factory.createIdentifier(d.displayName), typescript_1.default.factory.createIdentifier("displayName")), typescript_1.default.SyntaxKind.EqualsToken, typescript_1.default.factory.createStringLiteral(d.displayName))));
+    return insertTsIgnoreBeforeStatement(typescript_1.default.factory.createExpressionStatement(typescript_1.default.factory.createBinaryExpression(typescript_1.default.factory.createPropertyAccessExpression(typescript_1.default.factory.createIdentifier("exports."+d.displayName), typescript_1.default.factory.createIdentifier("displayName")), typescript_1.default.SyntaxKind.EqualsToken, typescript_1.default.factory.createStringLiteral(d.displayName))));
 }
 /**
  * Set a component prop description.
@@ -175,7 +175,7 @@ function createPropDefinition(propName, prop, options) {
  */
 function insertDocgenIntoGlobalCollection(d, docgenCollectionName, relativeFilename) {
     return insertTsIgnoreBeforeStatement(typescript_1.default.factory.createIfStatement(typescript_1.default.factory.createBinaryExpression(typescript_1.default.factory.createTypeOfExpression(typescript_1.default.factory.createIdentifier(docgenCollectionName)), typescript_1.default.SyntaxKind.ExclamationEqualsEqualsToken, typescript_1.default.factory.createStringLiteral("undefined")), insertTsIgnoreBeforeStatement(typescript_1.default.factory.createExpressionStatement(typescript_1.default.factory.createBinaryExpression(typescript_1.default.factory.createElementAccessExpression(typescript_1.default.factory.createIdentifier(docgenCollectionName), typescript_1.default.factory.createStringLiteral(`${relativeFilename}#${d.displayName}`)), typescript_1.default.SyntaxKind.EqualsToken, typescript_1.default.factory.createObjectLiteralExpression([
-        typescript_1.default.factory.createPropertyAssignment(typescript_1.default.factory.createIdentifier("docgenInfo"), typescript_1.default.factory.createPropertyAccessExpression(typescript_1.default.factory.createIdentifier(d.displayName), typescript_1.default.factory.createIdentifier("__docgenInfo"))),
+        typescript_1.default.factory.createPropertyAssignment(typescript_1.default.factory.createIdentifier("docgenInfo"), typescript_1.default.factory.createPropertyAccessExpression(typescript_1.default.factory.createIdentifier("exports."+d.displayName), typescript_1.default.factory.createIdentifier("__docgenInfo"))),
         typescript_1.default.factory.createPropertyAssignment(typescript_1.default.factory.createIdentifier("name"), typescript_1.default.factory.createStringLiteral(d.displayName)),
         typescript_1.default.factory.createPropertyAssignment(typescript_1.default.factory.createIdentifier("path"), typescript_1.default.factory.createStringLiteral(`${relativeFilename}#${d.displayName}`)),
     ]))))));
@@ -199,7 +199,7 @@ function setComponentDocGen(d, options) {
     var _a;
     return insertTsIgnoreBeforeStatement(typescript_1.default.factory.createExpressionStatement(typescript_1.default.factory.createBinaryExpression(
     // SimpleComponent.__docgenInfo
-    typescript_1.default.factory.createPropertyAccessExpression(typescript_1.default.factory.createIdentifier(((_a = d.expression) === null || _a === void 0 ? void 0 : _a.getName()) || d.displayName), typescript_1.default.factory.createIdentifier("__docgenInfo")), typescript_1.default.SyntaxKind.EqualsToken, typescript_1.default.factory.createObjectLiteralExpression([
+        typescript_1.default.factory.createPropertyAccessExpression(typescript_1.default.factory.createIdentifier("exports."+(((_a = d.expression) === null || _a === void 0 ? void 0 : _a.getName()) || d.displayName)), typescript_1.default.factory.createIdentifier("__docgenInfo")), typescript_1.default.SyntaxKind.EqualsToken, typescript_1.default.factory.createObjectLiteralExpression([
         // SimpleComponent.__docgenInfo.description
         typescript_1.default.factory.createPropertyAssignment(typescript_1.default.factory.createStringLiteral("description"), typescript_1.default.factory.createStringLiteral(d.description)),
         // SimpleComponent.__docgenInfo.displayName
